src/app/app.component.ts

```diff
@@ -67,16 +67,26 @@ export class AppComponent implements OnInit {
 
   // get weather coordinats
   getWeatherCoordinates(latitude: number, longitude: number): void {
+    console.log('Latitude:', latitude, 'Longitude:', longitude);
+
+    const weatherInfo = {
```

Comment:
It here a reason why we have a `WeatherInfo` obejct containing coordinates here, when we already have weatherCoordinates in the scope.

Reply:
I didn't really need to write it down, because I was supposed to see the weather data to see if it matched the coordinates. So I created an object having this information.